vaadin-testbench-core/pom.xml

```diff
@@ -87,12 +117,106 @@
         </resources>
 
         <plugins>
+            <plugin>
```

Comment:
There could actually be a note here that the kotlin plugin needs to be before the maven-compiler-plugin so that the kotlin files are compiled before the java compiler runs so that no-one moves the definitions around as a mistake.

Another would be to set the phase for kotlin so that it always executes before the java compileer.

Reply:
I would prefer adding a comment that explains the need to have kotlin plugin before maven-compiler-plugin, rather than using a phase different from `compile` for the kotlin plugin